app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PagesController < ApplicationController
+	def index
```

Comment:
Looks like your tab index is still set to four (4) spaces. Update your project to use two (2).